src/libraries/System.Speech/src/System.Speech.csproj

```diff
@@ -1,27 +1,19 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <TargetFrameworks>$(NetCoreAppCurrent)-windows;netcoreapp3.1-windows;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-windows;netcoreapp3.1-windows;netcoreapp3.1;netstandard2.0</TargetFrameworks>
```

Comment:
why doesn't this have `$(NetCoreAppCurrent)`? We seem to do that elsewhere

Reply:
You don't follow that same policy on 

src/libraries/System.Security.Principal.Windows/src/System.Security.Principal.Windows.csproj

There you added both `$(NetCoreAppCurrent)` and `netcoreapp3.1`.

Reply:
True. I think for Principal.Windows I chose to add `$(NetCoreAppCurrent)` configuration to the source so that in case another rid-less `$(NetCoreAppCurrent)` project references Principal.Windows, it will get the `$(NetCoreAppCurrent)` reference assembly. Without adding the configuration, the `netcoreapp3.1` asset would be chosen.

It shouldn't really matter though as the API set is identical but I felt like it's better to guarantee that `$(NetCoreAppCurrent)` reference assemblies are always chosen if possible over others.